apigw-resource-policy/README.md

```diff
@@ -0,0 +1,58 @@
+# Amazon API Gateway REST API with Resource Policy
+
+The SAM template deploys an Amazon API Gateway REST API endpoint that uses a Resource Policy for access control. The Resource Policy explicitly blacklists a list of IP ranges and also explicitly whitelists another list of IP ranges. If allowed by the Resource Policy, API Gateway will forward the request to a Lambda function which will output the received event object.
+
+Note: when deploying this pattern, *CAPABILITY_IAM* is required.
+
+Learn more about this pattern at Serverless Land Patterns: [https://serverlessland.com/patterns/apigw-resource-policy](https://serverlessland.com/patterns/apigw-resource-policy)
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd apigw-resource-policy
+    ```
+3. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam deploy -g
+    ```
+1. During the prompts:
+    * Enter a stack name
+    * Select the desired AWS Region
+    * Allow SAM to create roles with the required permissions if needed.
```

Comment:
you are missing instructions about the Parameter IpRangeBlacklist [10.20.30.40, 1.0.0.0/16] and Parameter IpRangeWhitelist [10.20.30.41, 1.1.0.0/16]

Reply:
Thanks a lot for the comments @mavi888 . I just added a pull request with the fixes.